api/node/routes.go

```diff
@@ -279,10 +272,10 @@ func PeerInfo(c *gin.Context) {
 	if !ok {
 		c.JSON(
 			http.StatusInternalServerError,
```

Comment:
Maybe remove the struct from routes_test.go? The one with the TODO, also the TODO on L352 should be fixed. Thanks.

Reply:
done